<?xml version="1.0"?>
<launch>

    <node pkg="joy" name="joy_node" type="joy_node">
        <param name="device" value="/dev/input/js0"/>
    </node>

    <node pkg="om_cart" name="om_manual" type="om_manual"/>

</launch>
